server.py

```diff
@@ -567,6 +588,13 @@ def get(self, ip, port):
 		i, server = self.getWithIndex(ip, port)
 		return server
 
+	def checkDuplicate(self, other_server):
```

Comment:
`isDuplicate` would be a more accurate name, which is unfortunately lexically similar to `is_duplicate`.

Reply:
I prefer `check` here for reasons I can't really explain.